### PR TITLE
Updates for podman

### PR DESCRIPTION
**Pull request by luisabv**
Update docker commands so that the syntax works for
both docker and podman.

Also add a try when updating the soft links for windows
so it doesn't fail if they are already fixed.

### CODE REVIEW

**Comment by luisabv** [comment]
@mishas , could you take a look to the changes and review?

File: dazel.py
Comment (mishas) [inline dazel.py:201]: This exception is a bit too broad. What exception do you expect here? can you catch the specific thing?
Also - besides pass please also explain why it's ok to ignore the exception here.

Comment (luisabv) [reply dazel.py]: Yes, this was badly done. Fixed now.

File: dazel.py
Comment (mishas) [inline dazel.py:201]: Actually, can you maybe put the try/except around the `output = self._run_command(command).strip()` line to make it clearer what it does?

Also - they way you did it now, if the first symlink exists, but the rest don't, it'll skip creating the other ones, which is not what you want.

Comment (luisabv) [reply dazel.py]: I moved inside the for loop, you are right about that we shall not skip the other links just because the first failed.
Regarding try/except around output, it is not possible if we fail here we cannot continue doing the other steps since they depend on the output.
But good comment!

File: dazel.py
Comment (mishas) [inline dazel.py:202]:
```suggestion
            try:
                output = self._run_command(command).strip()
            except subprocess.CalledProcessError:
                logger.info("INFO: Skipping fixing symlink, it already exists.")
            else:
                drive = pathlib.PureWindowsPath(path.drive)
                local_directory = drive.joinpath(pathlib.PurePosixPath(output))
                path.unlink()
                path.symlink_to(local_directory, target_is_directory=True)
```

**Review by mishas — COMMENTED** [review]
Apart for the one comment, i'm still waiting on some testing on our end, will approve once those are done.

Please ping me with a reminder in ~1 week if you don't hear back from me.

File: dazel.py
Comment (luisabv) [reply dazel.py]: I have updated according to your comments now and tested.

Comment (luisabv) [reply dazel.py]: @mishas . Ping ...

**Comment by luisabv** [comment]
ping!

**Review by mishas — APPROVED** [review]

**Pull request merged**
